CHANGELOG.md

```diff
@@ -41,6 +41,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * Templates now support the `>=`, `>`, `<=`, and `<` relational operators for
   `Integer` types.
 
+* `jj split` now accepts `--second` to put specified paths into the second
+  commit instead of the first.
```

Comment:
So `jj spit --second foo` is equivalent to `jj split ~foo`? Maybe we should just advertise the [fileset](https://martinvonz.github.io/jj/latest/filesets/) language more?

Reply:
yeah seriously!  I had no idea it takes a files because it says `PATHS` in the help string

Reply:
I will mutate this PR into a doc improvement

Reply:
> So `jj spit --second foo` is equivalent to `jj split ~foo`? Maybe we should just advertise the [fileset](https://martinvonz.github.io/jj/latest/filesets/) language more?

**Doh!**
And here I am in `jj-fzf split-files`, explicitly listing all files "not foo" in a bash array... Have to fix my act now...

OTOH, this does indicate that this fileset feature is totally underadvertised (if that was a word)

Reply:
Alright, I have adjusted this PR to make `split` explicit that it is using filesets.  That said, I left an `--invert-fileset` option, because simply doing `jj split ~file1 ~file2` will not do what people want (it will put parse as `~file1 | ~file2`, which is equivalent to `all()`).  Also, the tab completion aspect of using paths and then saying `--invert-fileset` is really helpful (as soon as you type `~` tab completion stops working).

Reply:
In almost half the cases where I use jj split interactively, I'd like to have an --invert option but for the hunks/diffs (to select diffs for the second instead of the first commit), not just filesets. Would it make sense for others to have a more general `--invert` option that applies to filesets *and* interactive splitting, instead of adding `--invert-fileset` today and `--invert-hunks` tomorrow?

Reply:
Yes, I was thinking the same thing in my last round of comments. I didn't say anything because the comment is irrelevant if we decide to just add more documentation pointing out that filesets are a thing :) But if we do want a way to invert the selection, I agree that it probably should apply to interactively selected hunks too.

Reply:
alright, I have switch this to being a pure doc PR and will move the `invert` discussion into a separate PR.